tutorial/conan_repositories/setting_up_conan_remotes/artifactory/artifactory_ce_cpp.rst

```diff
@@ -17,20 +17,56 @@ Running Artifactory CE
 
 There are several ways to run Artifactory CE:
 
-* **Running from a Docker image:**
+Running from a Docker image
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As of Artifactory 7.77, it is mandatory to use an additional SQL database such as
+PostgreSQL, or configure the ``allowNonPostgresql: true`` flag in the ``system.yaml`` file
+if you wish to use Derby or another database. You can pass that information via
+environment variables directly in the command line:
+
+1. **Using Derby** (with the flag ``allowNonPostgresql: true``):
 
 .. code-block:: bash
-    
-    $ docker run --name artifactory -d -p 8081:8081 -p 8082:8082 releases-docker.jfrog.io/jfrog/artifactory-cpp-ce:7.63.12
 
-* **Download and run from zip file**. The `Download Page <https://conan.io/downloads.html>`_ has
-  a link for you to follow. When the file is unzipped, launch Artifactory by double clicking
-  the artifactory.bat on Windows or artifactory.sh script in the *app/bin* subfolder,
-  depending on the OS. Artifactory comes with JDK bundled, please `read Artifactory
-  requirements <https://jfrog.com/help/r/jfrog-installation-setup-documentation/system-requirements>`_.
+    $ docker run --name artifactory -d -e JF_SHARED_DATABASE_TYPE=derby -e JF_SHARED_DATABASE_ALLOWNONPOSTGRESQL=true -p 8081:8081 -p 8082:8082 releases-docker.jfrog.io/jfrog/artifactory-cpp-ce:7.90.10
+
+2. **Launching PostgreSQL along with Artifactory**
+
+.. code-block:: bash
+
+    $ docker run --name postgres -itd -e POSTGRES_USER=artifactory -e POSTGRES_PASSWORD=password -e POSTGRES_DB=artifactorydb -p 5432:5432 library/postgres
```

Comment:
in both cases, it is generally ill advised to pass secrets via the command line invocations, this prints them to the console history file, and the `-e xxx==yyy` arguments are also visible to anybody inspecting running processes IIRC.

Writing them to a file and passing them to docker via `--env-file` may be more secure: https://docs.docker.com/reference/cli/docker/container/run/#env

Alternatively - we may wish to provide a sample docker compose file that would bring up both servers

Reply:
I wanted to keep things simple, in fact, this part is exactly the same that what you can find in the Artifactory documentation: https://jfrog.com/help/r/jfrog-installation-setup-documentation/install-artifactory-single-node-with-docker 
Maybe we can add a warning about it?